Sources/CircularProgress/protocol/MouseTrackable.swift

```diff
@@ -0,0 +1,25 @@
+#if os(macOS)
+import Cocoa
+/**
+Makes it possible to utilize mouseOver/mouseOut functionality on a view (Supports custom bezier path)
+*/
+protocol MouseTrackable:class{
+	var path:CGPath? {get}
+}
+extension MouseTrackable where Self:NSView{
+	/**
+	* Creates a new tracking area
```

Comment:
It also removes existing tracking areas. That should be mentioned too.

Reply:
Will do

Reply:
✅